tests/normal.t.pl

```diff
@@ -28,10 +28,10 @@ sub runTests
     my $hardlinks = 1;
     if($mode eq 'standard')
     {
-        &mount("--standard");
+        &mount("--standard -o direct_io");
     } elsif($mode eq 'paranoia')
     {
-        &mount("--paranoia");
+        &mount("--paranoia -o direct_io");
```

Comment:
From https://github.com/freebsd/freebsd/blob/stable/11/sys/fs/fuse/fuse_kernel.h#L117 :
`FOPEN_KEEP_CACHE: don't invalidate the data cache on open`

However `FOPEN_KEEP_CACHE` does not seem to be implemented at all :
https://github.com/freebsd/freebsd/blob/stable/11/sys/fs/fuse/fuse_node.c#L270

Certainly there is something like the following missing :
```
if (!(fuse_open_flags & FOPEN_KEEP_CACHE)) {
	fuse_io_invalbuf(vp, td);
}